src/app.js

```diff
@@ -2,26 +2,80 @@
 //We've got some basic info about Karen's home
 //Debug the type of data provided
 //Return the types concatenated in a single variable
+function moreAboutHome(address,distanceFromTown, hasNeighbours){
+    let typeOfAddress = typeof address;
```

Comment:
Why dont you use const keyword instead of let. Is there any specific reason to use the let keyword instead of Const